vme/src/szonelog.h

```diff
@@ -0,0 +1,105 @@
+#pragma once
```

Comment:
For part 1 of the split PR, you want this file and with code removal from cpp file